spider_robot/spider_robot.ino

```diff
@@ -23,14 +23,22 @@
 
 // modified by Regis for spider project, 2015-09-26
 // add remote control by HC-06 bluetooth module
-
+//
+// Change to use SerialCommands library.
+//   Allows easy selection of serial port for bluetooth module.
+//   SerialCommands is available from Arduino library.
+//   Ebony 2018-07-05
 
 /* Includes ------------------------------------------------------------------*/
 #include <Servo.h>    //to define and control servos
 #include <FlexiTimer2.h>//to set a timer to manage all servos
 // RegisHsu, remote control
-#include <SerialCommand.h>
-SerialCommand SCmd;   // The demo SerialCommand object
+#include <SerialCommands.h>
+
+/* Serial Commands------------------------------------------------------------*/
+Stream & CmdSerial = Serial1;
+char serial_command_buffer_[32];
+SerialCommands serial_commands_(&Serial1, serial_command_buffer_, sizeof(serial_command_buffer_), "\r\n", " ");
```

Comment:
This variable name ("serial_commands_") doesn't match the scheme used for other variables (e.g. "CmdSerial", "SCmd"). How about "CommandProcessor"?